app/components/ChartsMain.tsx

```diff
@@ -117,16 +124,19 @@ export const ChartsMain = () => {
         </div>
         <div>
           These coins:{" "}
-          {combinedChartCoins.map((c) => {
-            return <div key={c.coinName}>{c.coinName}</div>;
+          {combinedChartCoins?.map((c) => {
+            return <div key={c?.coinName}>{c?.coinName}</div>;
```

Comment:
do you really need the ? inside the map?

Reply:
https://www.loom.com/share/7f47f5df7cd5467c9be0a4023698c7da?from_recorder=1&focus_title=1